.drone.yml

```diff
@@ -4,7 +4,7 @@ workspace:
 
 pipeline:
   build_binary:
-    image: golang:1.8.1
```

Comment:
probably shouldn't change this, unless you've tested it compiles and works properly

Reply:
i haven't... i'll change it back